examples/widgets/chart/lv_chart_example_8.c

```diff
@@ -0,0 +1,115 @@
+#include "../../lv_examples.h"
```

Comment:
Please rename the file to `lv_example_chart_8.c` for consistency. I know I'm being a bit picky - sorry. :slightly_smiling_face: